content/docs/cucumber/state.md

```diff
@@ -134,9 +149,13 @@ compile group: 'io.cucumber', name: 'cucumber-picocontainer', version: '{{% vers
 
 There is no documentation yet, but the code is on [GitHub](https://github.com/cucumber/cucumber-jvm/tree/master/picocontainer).
 For more information, please see [sharing state using Picocontainer](http://www.thinkcode.se/blog/2017/04/01/sharing-state-between-steps-in-cucumberjvm-using-picocontainer).
+{{% /block %}}
 
-### Spring
+{{% block "ruby,javascript" %}} Picocontainer is a Dependency Injection framework for JVM languages. {{% /block %}}
```

Comment:
Should it be 'PicoContainer' instead of 'Picocontainer'?

Reply:
It should be spring, actually. Thanks for pointing that out!

Reply:
Yes, thanks!